LICENSE.txt

```diff
@@ -0,0 +1,11 @@
+Servus - A small set of network oriented utilities in C++ including a zeroconf implementation.
+
+Servus is licensed under the LGPL version 3, unless noted otherwise, e.g., for external dependencies. See file COPYING.txt for the full license. External dependencies are either LGPL or BSD-licensed. See file ACKNOWLEDGEMENTS.txt and AUTHORS.txt for further details.
```

Comment:
I kept the original copyright notice of every file, so I don't think this is a concern.
Regarding the authors I have found @eile, Cedric, @tribal-tec and @bilgili, but no one else. John has contributed to the CMake subtree, but that should be considered a separate repo, shouldn't it.